app/controllers/schools/add_participants_controller.rb

```diff
@@ -2,87 +2,25 @@
 
 module Schools
   class AddParticipantsController < ::Schools::BaseController
-    FORM_SESSION_KEY = :add_participant_form
-    FORM_PARAM_KEY = :schools_add_participant_form
+    include Multistep::Controller
 
     skip_after_action :verify_authorized
-    before_action :ensure_form_present, except: :start
     before_action :set_school_cohort
 
-    helper_method :add_participant_form
-
-    def start
-      session.delete(FORM_SESSION_KEY)
-      session[FORM_SESSION_KEY] = {
-        school_cohort_id: @school_cohort.id,
-      }
-      redirect_to action: :show, step: :type
-    end
-
-    def show
-      render current_step
-    end
-
-    def update
-      if add_participant_form.valid?(current_step)
-        add_participant_form.record_completed_step current_step
-        store_form_in_session
-        redirect_to action: :show, step: step_param(add_participant_form.next_step(current_step))
-      else
-        render current_step
-      end
-    end
-
-    def complete
-      @participant_profile = add_participant_form.save!
-      @type = add_participant_form.type
-
-      session.delete(FORM_SESSION_KEY)
-    end
+    form AddParticipantForm, as: :add_participant_form
+    escape_route { schools_cohort_participants_path }
```

Comment:
Does this not reintroduce the issue in #516 ?

Reply:
Yes, there will be a merge conflict to resolve on that. I'd leave it as is for now and will update the PR when the other one is merged.

Reply:
It is now corrected :)